packages/terra-paginator/CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 * Added
   * Added visually hidden text with meaningful contexts to help screen reader users understand when they navigate the pagination.
+* Changed
```

Comment:
nit: I am not sure whats being followed in this repo but do we need to add a new line before this?

Reply:
Fixed: https://github.com/cerner/terra-core/pull/3897/commits/80ad9216dd672e1a59665b19496b871f73b1197a